opentreemap/treemap/js/src/CanopyFilterControl.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+var $ = require('jquery'),
+    _ = require('lodash'),
+    L = require('leaflet'),
+    Bacon = require('baconjs');
+
+require('ionRangeSlider');
+
+var canopyFilterTmpl = _.template($('#canopy-filter-tmpl').html());
+
+var CanopyFilterControl = L.Control.extend({
+    options: {
+        position: 'bottomleft'
+    },
+
+    initialize: function() {
+        this._changeBus = new Bacon.Bus();
+        this.tilerArgsProp = this._changeBus.toProperty({
+            category: 'Municipality',
```

Comment:
I wasn't aware that column existed. In that case, I'll update the code based on your suggestions.